src/app/features/tech-record/components/tech-record-summary-changes/tech-record-summary-changes.component.ts

```diff
@@ -0,0 +1,207 @@
+import {
+  ChangeDetectionStrategy, Component, OnDestroy, OnInit,
+} from '@angular/core';
+import { ActivatedRoute, Router } from '@angular/router';
+import { GlobalErrorService } from '@core/components/global-error/global-error.service';
+import { HGVAxles } from '@dvsa/cvs-type-definitions/types/v3/tech-record/get/hgv/complete';
+import { PSVAxles } from '@dvsa/cvs-type-definitions/types/v3/tech-record/get/psv/skeleton';
+import { TRLAxles } from '@dvsa/cvs-type-definitions/types/v3/tech-record/get/trl/complete';
+import { TechRecordType } from '@dvsa/cvs-type-definitions/types/v3/tech-record/tech-record-verb';
+import { TechRecordGETHGV, TechRecordGETPSV, TechRecordGETTRL } from '@dvsa/cvs-type-definitions/types/v3/tech-record/tech-record-verb-vehicle-type';
+import { FormNode, FormNodeViewTypes } from '@forms/services/dynamic-form.types';
+import { VehicleSummary } from '@forms/templates/tech-records/vehicle-summary.template';
+import { vehicleTemplateMap } from '@forms/utils/tech-record-constants';
+import { VehicleTypes } from '@models/vehicle-tech-record.model';
+import { Actions, ofType } from '@ngrx/effects';
+import { Store } from '@ngrx/store';
+import { RouterService } from '@services/router/router.service';
+import { TechnicalRecordService } from '@services/technical-record/technical-record.service';
+import { State } from '@store/index';
+import {
+  clearAllSectionStates,
+  clearScrollPosition,
+  editingTechRecord,
+  selectTechRecordChanges,
+  selectTechRecordDeletions,
+  techRecord,
+  updateTechRecord,
+  updateTechRecordSuccess,
+} from '@store/technical-records';
+import {
+  Subject, combineLatest, take, takeUntil,
+} from 'rxjs';
+
+@Component({
+  selector: 'app-tech-record-summary-changes',
+  templateUrl: './tech-record-summary-changes.component.html',
+  styleUrls: ['./tech-record-summary-changes.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class TechRecordSummaryChangesComponent implements OnInit, OnDestroy {
+  destroy$ = new Subject<void>();
+
+  techRecord?: TechRecordType<'get'>;
+  techRecordEdited?: TechRecordType<'put'>;
+  techRecordChanges?: Partial<TechRecordType<'get'>>;
+  techRecordDeletions?: Partial<TechRecordType<'get'>>;
+  techRecordChangesKeys: string[] = [];
+
+  sectionState?: (string | number)[];
+  sectionsWhitelist: string[] = [];
+
+  constructor(
+    public store$: Store<State>,
+    public technicalRecordService: TechnicalRecordService,
+    public router: Router,
+    public globalErrorService: GlobalErrorService,
+    public route: ActivatedRoute,
+    public routerService: RouterService,
+    public actions$: Actions,
+  ) {}
+
+  ngOnInit(): void {
+    this.actions$.pipe(ofType(updateTechRecordSuccess), takeUntil(this.destroy$)).subscribe((vehicleTechRecord) => {
```

Comment:
I think it would be good to have this an a separate method, navigate to update record or something , more readable